frontend/src/components/getInvolved.tsx

```diff
@@ -6,49 +6,49 @@ import TutorTutee from "../assets/images/homepage_tutor_tutee_2.svg";
 
 const GetInvolved = () => {
     return (
-      <div className="bg-[#FBFAFC] p-24">
+      <div className="bg-[#FBFAFC] p-12 md:p-24">
         <div className="max-w-6xl mx-auto space-y-15">
-            <div className="flex items-center justify-center">
+            <div className="flex items-center justify-center pb-10">
                 <div className="border-b-2 border-[#E0D1ED]" style={{ width: "40px" }}></div>
                 <h1 className="font-interMedium mx-4 text-center" style={{ fontSize: "32px", color: "#1F3A68" }}>
                 GET INVOLVED
                 </h1>
                 <div className=" border-b-2 border-[#E0D1ED]" style={{ width: "40px" }}></div>
             </div>
 
-        <div className="flex flex-row flex-wrap justify-center">
-            <div className="flex flex-row justify-center">
-            <div className="basis-1/2 space-y-10 mb-24">
-                <h1 className=" font-inter font-semibold" style={{ fontSize: "32px"}}>For Tufts Students</h1>
-                <div className="text-left space-y-6 text-[#545353]" style={{ fontSize: "22px"}} >
-                    <p>This is a great opportunity for students to give back to their local community and is a rewarding experience</p>
-                    <p>You can sign up to be a tutor anytime through this form</p>
-                    <div className="mb-6"></div>
-                    <button className="rounded-full bg-[#1E3B68] p-6 text-white text-lg" type="button">Become a Tutor!</button>
+        <div className="flex flex-wrap">
+            <div className="flex flex-col lg:flex-row">
+                <div className="basis-full lg:basis-1/2 space-y-10 mb-24">
```

Comment:
Very nice work with the breakpoint classes. I am eventually going to go through and change the inline CSS to tailwind just to standardize the code.